src/support_sphere_py/src/support_sphere/models/public/people.py

```diff
@@ -0,0 +1,52 @@
+import uuid
+from typing import Optional
+
+from sqlmodel import Field, Relationship
+
+from support_sphere.models.base import BasePublicSchemaModel
+
+
+class People(BasePublicSchemaModel, table=True):
+
+    """
+    Represents an individual in the 'public.people' table.
+    An individual represented as a row in this table,
+    may or may NOT have an account in (auth.users) or profile in (user_profiles)
+
+    Attributes
+    ----------
+    id : uuid.UUID
+        The unique identifier for the individual, generated automatically using UUID.
+    user_id : uuid.UUID, optional
```

Comment:
```suggestion
    user_profile_id : uuid.UUID, optional
```

Since `user` refers to the `auth.users` table, let's name this to `user_profile_id` to be more clear.

Reply:
updated